content/en/security_platform/application_security/getting_started/ruby.md

```diff
@@ -19,30 +19,29 @@ further_reading:
 
 1. **Update your gem file to include the Datadog library**:
    ```
-   gem install ddtrace
+   gem 'ddtrace', '~> 1.0.0.beta1'
    ```
    For information about which language and framework versions are supported by the library, see [Compatibility][1].
 
 2. **Enable Application Security**, either in your code:
    ```
    require 'datadog/security'
-   Datadog::Security.configure do |c|
-     c.use :rails
-   end
-   # not needed if require 'ddtrace/auto_instrument' is used
    Datadog.configure do |c|
-     c.use :rails
+     c.appsec.instrument :rails
```

Comment:
This could also be `:sinatra` or `:rack` depending on the framework.

For frameworkless `:rack`, just as with the APM tracer, auto instrumentation is not possible, therefore the app needs to have the following lines manually added as well in `config.ru`:

```
use Datadog::Tracing::Contrib::Rack::TraceMiddleware
use Datadog::AppSec::Contrib::Rack::RequestMiddleware
```

Not sure how to best present that without being increasingly confusing.

Reply:
I guess a good question here is: is this document supposed to be as detailed as, for example, https://github.com/DataDog/dd-trace-rb/blob/master/docs/GettingStarted.md#rack?

If this is the end-all-be-all for appsec in Ruby, sure we can expand for all use cases.

Otherwise, I think it would make sense to point something like: "For other gems and frameworks, see <ins>link</ins>".

Reply:
Btw, APM Ruby Tracing uses this magic here to automatically publish our github markdown into official docs: https://github.com/DataDog/documentation/blob/ddfe8595e163615f671e4576dc75830c0a6a6d40/local/bin/py/build/configurations/pull_config.yaml#L196-L217

This gives us total ownership over the documentation.

Reply:
> If this is the end-all-be-all for appsec in Ruby, sure we can expand for all use cases.

Not long term, there should be documentation in the ddtrace repo (like there is for APM), but this does not exist yet and AFAIK this is all that exists in terms of AppSec setup doc apart from [troubleshooting](https://github.com/DataDog/documentation/pull/13002)

Reply:
I would suggest something simple such as:

```
  c.appsec.instrument :rails # can be :rack or :sinatra when applicable

...

# note: :rack needs manual configuration in `config.ru` such as:
#   use Datadog::Tracing::Contrib::Rack::TraceMiddleware
#   use Datadog::AppSec::Contrib::Rack::RequestMiddleware
```